judo-tatami-jsl-jsl2psm/src/main/epsilon/transformations/psm/modules/structure/transferObjectRelation.etl

```diff
@@ -258,3 +258,88 @@ rule CloneTransferObjectQueryRelationForDefaultTransferObjectType
         
         log.debug("Clone TransferObjectRelation for DefaultTransferObjectType: [" + t.name + "]");
 }
+
+
+///////////////////////////////////////////////////////////////
+// Transfer composite relation
+///////////////////////////////////////////////////////////////
+
+@abstract
+rule AbstractCreateTransferObjectEmbeddedRelationForTransferFieldDeclaration
+    transform s: JSL!TransferFieldDeclaration
+    to t: JUDOPSM!TransferObjectRelation {
+        guard: s.referenceType.isKindOf(JSL!TransferDeclaration)
+
+        t.name = s.name;
+        t.target = s.referenceType.getPSMEquivalent();
+
+        // t.embeddedCreate = not s.eContainer.isAbstract;
+        // t.embeddedUpdate = not s.eContainer.isAbstract;
+        // t.embeddedDelete = not s.eContainer.isAbstract;
+        
+		t.embedded = true;
+        // t.binding = s.equivalent("CreateContainmentFromField");
```

Comment:
Code in comments?

Reply:
There wil be rules which can be processed soon - this ticket is unfinished, only some parts are implemented, and Its a reminder where can and how this values.